Python/Product/VSCode/AnalysisVsc/LanguageServer.cs

```diff
@@ -140,7 +174,7 @@ public Task DidOpenTextDocument(JToken token)
            => _server.DidOpenTextDocument(token.ToObject<DidOpenTextDocumentParams>());
 
         [JsonRpcMethod("textDocument/didChange")]
-        public Task DidChangeTextDocument(JToken token)
+        public void DidChangeTextDocument(JToken token) 
```

Comment:
Is it a public API?

Reply:
Sort of. Call comes from VSC process over RPC. StreamJsonRPC handles is async or plain depending on the signature.

Reply:
I'm just thinking if this method can become async in future version of API